example-jobs/checkmarx-security-job-example.groovy

```diff
@@ -0,0 +1,18 @@
+import jenkins.automation.utils.CheckmarxSecurityJobBuilder
+
+def projectName ='foo'
+new CheckmarxSecurityJobBuilder(
+        name: "${projectName}-checkmarx",
+        description: "Sample checkmarx security job",
+        useOwnServerCredentials: false,
+        serverUrl: "http://awe-codescan-w-d01",
```

Comment:
let's not hard-code these serverUrl or groupId params; instead, we need to pass them in from the jobs; we can talk about making them environment variables or something else to abstract them, but they're internal/private data and consequently should not be in a public repo